cfme/fixtures/vm.py

```diff
@@ -1,44 +1,74 @@
 # -*- coding: utf-8 -*-
 import pytest
+from pytest_polarion_collect.utils import get_parsed_docstring
+from pytest_polarion_collect.utils import process_json_data
 from wrapanapi import VmState
 
 from cfme.exceptions import CFMEException
-from cfme.utils import ports
 from cfme.utils.generators import random_vm_name
-from cfme.utils.net import net_check
-from cfme.utils.wait import wait_for
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        '--no-assignee-vm-name',
+        action='store_true',
+        default=False,
+        help=(
+            'If passed no-assignee-vm-name, vm names will not contain test_case assignee names.'
+        )
+    )
+
+
+@pytest.hookimpl(trylast=True)
+def pytest_collection_modifyitems(session, config, items):
```

Comment:
Discussed offline - there's no collection modification happening here, it would be better to call process_json_data in `pytest_collection_finish`.